org.standardnotes.standardnotes.yml

```diff
@@ -0,0 +1,120 @@
+app-id: org.standardnotes.standardnotes
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+# Electron doesn't use a traditional locale format so separate-locales is useless.
+separate-locales: false
+command: start-standardnotes
+finish-args:
+  - '--device=dri'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-run/keyring'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+modules:
+  - name: yarn
+    buildsystem: simple
+    build-commands:
+      - 'cp -a * /app'
+    # Only used for building, so clean it up afterwards.
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://github.com/yarnpkg/yarn/releases/download/v1.22.4/yarn-v1.22.4.tar.gz
+        sha256: bc5316aa110b2f564a71a3d6e235be55b98714660870c5b6b2d2d3f12587fb58
+
+  - name: standardnotes
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: '/usr/lib/sdk/node14/bin:/app/yarn/bin:/run/build/standardnotes/flatpak-node/chromedriver/'
+      env:
+        # Don't add ELECTRON_CACHE
+        XDG_CACHE_HOME: /run/build/standardnotes/flatpak-node/cache
+        # ELECTRON_CACHE: '/run/build/standardnotes/flatpak-node/electron-cache/'
+        CHROMEDRIVER_SKIP_DOWNLOAD: 'true'
+        JOBS: max
+    build-commands:
+      # Retire this at some point; (1) create a new task by default and (2) use Yarn instead"
+      - "sed -i '/dist:linux/c\\    \\\"dist-linux\\\": \\\"npm run build &&
+        electron-builder --linux\\\",' package.json"
+      # - "find . -type f -name 'package-lock.json' -exec rm {} +"
+
+      # Use the offline mirror.
+      - 'HOME=$PWD yarn config --offline set yarn-offline-mirror
+        $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror'
+
+      # Translated from npm run setup
+      - yarn --offline --pure-lockfile
+      # - yarn --offline --pure-lockfile --cwd ./web
+      - yarn --offline --pure-lockfile --cwd ./app
+
+      # - 'npm --offline --save false ci
+      #   --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache'
+      # - 'npm --offline --save false --prefix ./web install
+      #   --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache'
+      # - 'npm --offline --save false --prefix ./app ci
+      #   --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache'
+
+      - . flatpak-node/electron-builder-arch-args.sh
```

Comment:
This line effectively does nothing, since each each command in `build-commands` is spawned in a new shell.

Reply:
I see. That's why there are separated by a semicolon in the example